packages/core/src/ceramic.ts

```diff
@@ -59,6 +66,9 @@ export interface CeramicConfig {
   networkName?: string;
   pubsubTopic?: string;
 
+  docBaseCacheLimit?: number;
```

Comment:
Can we call this just `docCacheLimit` since it applies to both base and commit caches?

Reply:
👍